crocket/src/main/java/com/crocket/model/interfaces/ITargetEvent.java

```diff
@@ -0,0 +1,10 @@
+package com.crocket.model.interfaces;
+
+import com.crocket.model.entity.Ball;
+import com.crocket.model.entity.Entity;
+
+public interface ITargetEvent {
+    public Entity getTarget();
```

Comment:
Should this maybe be a separate type `Target`?

Reply:
Removed the file